docs/user-guide/configuring.md

```diff
@@ -205,14 +205,14 @@ This defines two global variables, `var1` and `var2`. If you want to optionally
 /* global var1:false, var2:false */
 ```
 
-To configure global variables inside of a configuration file, use the `globals` key and indicate the global variables you want to use. Set each global variable name equal to `true` to allow the variable to be overwritten or `false` to disallow overwriting. For example:
+To configure global variables inside of a configuration file, use the `globals` key and indicate the global variables you want to use in a list. Additionally, you may specify whether or not the variable can be overwritten by appending `:true` or `:false` (the default) to the variable name. For example:
 
 ```json
 {
-    "globals": {
-        "var1": true,
-        "var2": false
-    }
+    "globals": [
```

Comment:
https://github.com/eslint/eslint/blob/master/lib/config.js#L211-L217

It seems to be parsed like one.